### PR TITLE
feat: release trigger example.

### PR DESCRIPTION
**Pull request by wdstar**
## Context

We add a release triggered job example.

## References

- https://github.com/screwdriver-cd/screwdriver/issues/823

### CODE REVIEW

**Review by s-yoshika — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+        image: node:10
+        steps:
+            - echo: |
+                echo 'this job is executed by a release event.'
+        requires: [~release]
+```
+
+In the above example, the `release-triggered-job` is started when you create a release.
+
+**Note**: if you create a release with a **new** tag, both of `tag-triggered-job` and `release-triggered-job` are executed.
```
Comment (sakka2) [inline README.md:30]: It is kind to add a link to `tag-triggered-job` example

**Review by sakka2 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (wdstar) [reply README.md:30]: Thanks. I added it.

**Review by tk3fftk — APPROVED** [review]
LGTM:+1:

**Pull request merged**